lib/gui/src/easing_animator.rs

```diff
@@ -1,12 +1,14 @@
 //! EasingAnimator examples.
 
+use crate::prelude::*;
+
 use wasm_bindgen::prelude::*;
-use crate::animation::easing::*;
-use crate::animation::animator::easing::EasingAnimator;
-use crate::system::web::create_element;
-use crate::system::web::NodeInserter;
-use crate::system::web::AttributeSetter;
-use crate::system::web::StyleSetter;
+use basegl::animation::easing::*;
+use basegl::animation::animator::easing::EasingAnimator;
+use basegl::system::web::create_element;
+use basegl::system::web::NodeInserter;
+use basegl::system::web::AttributeSetter;
+use basegl::system::web::StyleSetter;
```

Comment:
These imports should be reorganized.

Reply:
Ok, doing it, but it was not my code originally. CC @notdanilo  - take care about sorting imports please.